spec/views/articles_spec.rb

```diff
@@ -54,4 +54,14 @@ def create_comment(parent_id = nil)
     expect(rendered).to have_text("Originally published at")
     expect(rendered).to have_text("example.com")
   end
+
+  it "shows a note about the canonical URL after edit" do
+    allow(article1).to receive(:canonical_url).and_return("https://example.com/lamas")
+    allow(article1).to receive(:published_at).and_return(DateTime.strptime("1", "%s"))
```

Comment:
I think you can use `Time.zone.at(0)` here if you want to use the "beginning of time"